Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -180,6 +197,42 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
+def is_tim_content(packs_in_pr: set[str]) -> bool:
+    """
+    :param packs_in_pr: packs that are part of the PR
+    :return: returns True or False if tim reviewer needed
+    """
+    for pack in packs_in_pr:
+        pack_object = BaseContent.from_path(CONTENT_PATH / pack)
+        try:
+            if isinstance(pack_object, Pack):
+                for integration in pack_object.content_items.integration:
+                    if integration.is_feed:
+                        return True
```

Comment:
This logic is incorrect. Imagine a PR where a non-TIM script was changed, this code will extract the pack and will check if there's an integration in this pack of TIM type and will return true. This is obviously wrong cause we don't want to involve TIM owner in such a case, we will only if the file changed belongs to an integration of type feed.